### PR TITLE
Profiling wrappers

### PR DESCRIPTION
**Pull request by aelanman**
Some updated wrappers for performing and summarizing profiling jobs across the parameter space. Designed for SLURM.

### CODE REVIEW

File: pyuvsim/simsetup.py [hunk 1]
```diff
@@ -621,6 +632,7 @@ def uvdata_to_config_file(uvdata_in, param_filename=None, telescope_config_name=
 
     if param_filename is None:
         param_filename = check_file_exists_and_increment(os.path.join(path_out, 'obsparam.yaml'))
+        param_filename = os.path.basename(param_filename)
```
[the remark this answers is not in the document]
Comment (aelanman) [reply pyuvsim/simsetup.py]: That's interesting... I put that line in to fix a bug that I uncovered while trying to cover the line above it. I'll see what I can do.

File: pyuvsim/tests/test_simsetup.py [hunk 2]
```diff
@@ -294,12 +293,24 @@ def test_mock_catalogs():
...
+    nt.assert_raises(KeyError, pyuvsim.simsetup.create_mock_catalog, time, 'invalid_catalog_name')
+
     for arr in arrangements:
         radec_catalog = pyuvsim.simsetup.read_text_catalog(os.path.join(SIM_DATA_PATH,
-                                                           'test_catalogs', text_catalogs[arr]))
-        print(arr)
+                                                                        'test_catalogs', text_catalogs[arr]))
         nt.assert_true(np.all(radec_catalog == cats[arr]))
 
+    cat, mock_kwds = pyuvsim.simsetup.create_mock_catalog(time, 'random', save=True)
```
[the remark this answers is not in the document]
Comment (aelanman) [reply pyuvsim/tests/test_simsetup.py]: I'll add that!

File: scripts/many_profile.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#!/bin/env python
+
```
[the remark this answers is not in the document]
Comment (aelanman) [reply scripts/many_profile.py]: Done in my last commit.

File: pyuvsim/tests/test_simsetup.py [hunk 4]
```diff
@@ -294,12 +319,24 @@ def test_mock_catalogs():
                      'triangle': 'mock_triangle_2458098.27471.txt',
                      'zenith': 'mock_zenith_2458098.27471.txt'}
```
Comment (bhazelton) [inline pyuvsim/tests/test_simsetup.py]: now that you have the seed, you could test the random array in the same way, specifying the seed to be the same as in the saved file.

Comment (aelanman) [reply pyuvsim/tests/test_simsetup.py]: I've added that in the latest commit.

**Pull request merged**